documentation/src/main/asciidoc/user_guide/integrations_hibernate_cache.adoc

```diff
@@ -0,0 +1,506 @@
+However, Infinispan also provides a configuration designed for local, single node, environments.
+To enable that configuration, set `hibernate.cache.infinispan.cfg` to `org/infinispan/hibernate/cache/commons/builder/infinispan-configs-local.xml` value.
+You can find more about the configuration check the <<default_local_configuration,local configuration>> section.
+
+A simple tutorial showing how to use Infinispan as Hibernate cache provider in a standalone application can be found
+link:https://github.com/infinispan/infinispan-simple-tutorials/tree/master/hibernate-cache/local[here].
+
+===== Single-Node Standalone Spring Application
+
+Using Hibernate within Spring applications is a very common use case.
+In this section you will learn what you need to do configure Hibernate within Spring to use Infinispan as second-level cache provider.
+
+As in the previous case, start by making sure that Hibernate Infinispan Cache provider is available in the classpath.
+Then, modify `application.properties` file to contain:
+
+    # Use Infinispan second level cache provider
+    spring.jpa.properties.hibernate.cache.region.factory_class=infinispan
+    # 
+    # Force using local configuration when only using a single node.
+    # Otherwise a clustered configuration is loaded.
+    spring.jpa.properties.hibernate.cache.infinispan.cfg=org/infinispan/hibernate/cache/commons/builder/infinispan-configs-local.xml
+
+By default when running standalone, the Infinispan second-level cache provider uses an Infinispan configuration that’s designed for clustered environments.
+However, Infinispan also provides a configuration designed for local, single node, environments.
+To enable that configuration, set `spring.jpa.properties.hibernate.cache.infinispan.cfg` to `org/infinispan/hibernate/cache/commons/builder/infinispan-configs-local.xml` value.
+You can find more about the configuration check the <<default_local_configuration,local configuration>> section.
+
+A simple tutorial showing how to use Infinispan as Hibernate cache provider in a Spring application can be found
+link:https://github.com/infinispan/infinispan-simple-tutorials/tree/master/hibernate-cache/spring-local[here].
+
+===== Single-Node WildFly Application
+
+In WildFly, Infinispan is the default second level cache provider for JPA/Hibernate.
+This means that when using JPA in WildFly, region factory is already set to `infinispan`.
+Infinispan's configuration is located in WildFly's `standalone.xml` file.
+It follows the same settings explained in <<default_local_configuration,local configuration>> section.
+
+Several aspects of the Infinispan second level cache provider can be configured directly in `persistence.xml`.
+This means that some of those tweaks do not require changing WildFly's `standalone.xml` file.
+You can find out more about these changes in the <<configuration_properties, configuration properties>> section.
```

Comment:
is the link working for you? I had to change to `_configuration_properties` to make it work.

Reply:
edit: similar issue for `default_local_configuration` and `default_cluster_configuration`